tests/cupy_tests/random_tests/test_sample.py

```diff
@@ -114,7 +114,7 @@ class TestRandintDtype(unittest.TestCase):
     def test_dtype(self, dtype):
         size = (1000,)
         low = numpy.iinfo(dtype).min
-        high = numpy.iinfo(dtype).max + 1
+        high = numpy.iinfo(dtype).max
```

Comment:
Hmmm, this change the test, Maybe need at least a new xfail here since that should work?

Reply:
Indeed. Have to xfail for numpy 1.26 + remove short ufunc loops here as well:
https://github.com/cupy/cupy/pull/8323/commits/7217c8dc2ff9269c4aeffaa31945ef8c13b8705c